addons/pos_restaurant/static/src/js/tours/pos_restaurant.js

```diff
@@ -199,7 +199,7 @@ odoo.define('pos_reataurant.tour.synchronized_table_management', function (requi
     steps = steps.concat(open_table('T2'));
 
     // Test transfering an empty order
-    steps = steps.concat(transfer_order_to_table('T4', '2'));
```

Comment:
you can remove this commit as it has been changed from lap and lca instructions in this commit: https://github.com/odoo/odoo/commit/73d877eef248d1ec4e22a03e966f0d8b91012389

Reply:
ok, done!